directory/src/main/java/org/ecclesia/directory/domain/Person.java

```diff
@@ -1,15 +1,18 @@
 package org.ecclesia.directory.domain;
 
+import java.io.Serializable;
 import javax.persistence.*;
-import lombok.Data;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
 
 /**
  * Represents one person. Data transfer object.
  */
 @Entity
 @Table(name = "person")
-@Data
-public class Person {
+@Getter @Setter @NoArgsConstructor
```

Comment:
**Reporter**: Spotbugs
**Rule**: SE_NO_SERIALVERSIONID
**Severity**: INFO
**File**: org/ecclesia/directory/domain/Person.java L14
**Source**: org.ecclesia.directory.domain.Person

Class is Serializable, but doesn&#39;t define serialVersionUID&#10;&#10;&#10;  &lt;p&gt; This class implements the &lt;code&gt;Serializable&lt;/code&gt; interface, but does&#10;  not define a &lt;code&gt;serialVersionUID&lt;/code&gt; field.&amp;nbsp;&#10;  A change as simple as adding a reference to a .class object&#10;    will add synthetic fields to the class,&#10;   which will unfortunately change the implicit&#10;   serialVersionUID (e.g., adding a reference to &lt;code&gt;String.class&lt;/code&gt;&#10;   will generate a static field &lt;code&gt;class$java$lang$String&lt;/code&gt;).&#10;   Also, different source code to bytecode compilers may use different&#10;   naming conventions for synthetic variables generated for&#10;   references to class objects or inner classes.&#10;   To ensure interoperability of Serializable across versions,&#10;   consider adding an explicit serialVersionUID.&lt;/p&gt;&#10;


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1169977716>*